### PR TITLE
No sec ban borg

### PR DESCRIPTION
**Pull request by JamieD1**
People that are sec banned can no longer be sec borgs.

### CODE REVIEW

**Comment by alexkar598** [comment]
Very little overlap between security borgs and security but ok

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -822,4 +822,7 @@
 	if(CONFIG_GET(flag/disable_secborg))
 		to_chat(user, "<span class='warning'>Nanotrasen policy disallows the use of weapons of mass destruction.</span>")
 		return FALSE
+	if(is_banned_from(R.ckey, "Security Officer"))
+		to_chat(user, "<span class='warning'>Nanotrasen has disallowed this unit from becoming this type of module</span>")
```
Comment (Bibby0110) [inline code/game/objects/items/robot/robot_upgrades.dm]:
```suggestion
		to_chat(user, "<span class='warning'>Nanotrasen has disallowed this unit from becoming this type of module.</span>")
```

**Comment by maxion12345** [comment]
But...cyborgs don't follow spacelaw...

**Comment by Cartlord** [comment]
As far as I know this was made because some Secbanned people use Secborgs to evade

**Pull request merged**